chingsley/ireporter-react
# #164041103 Implement admin report management feature

#### What does this PR do?
It implements the feature to enable an admin to fetch all reports and to change the status of any report.

#### Description of Task to be completed?
- design the admin report management page (the admin dashboard)
- fetch and render all reports on the page
- implement the feature to enable admin change the status of a report

#### How should this be manually tested?
- pull the branch
- ensure the necessary dependencies are installed
- sign in as an admin to view the admin page and examine the status editing feature

#### Any background context you want to provide?
- only users with admin credentials can view the admin dashboard or change the status of a report

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/164041103

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
